<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>姓名(&amp;N)</string>
         </property>
         <property name="buddy">
          <cstring>name_lineedit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sex_label">
         <property name="text">
          <string>性别(&amp;S)</string>
         </property>
         <property name="buddy">
          <cstring>sex_comboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="age_label">
         <property name="text">
          <string>年龄(&amp;A)</string>
         </property>
         <property name="buddy">
          <cstring>age_spinBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mail_label">
         <property name="text">
          <string>邮箱(&amp;M)</string>
         </property>
         <property name="buddy">
          <cstring>mail_lineedit</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="name_lineedit"/>
       </item>
       <item>
        <widget class="QComboBox" name="sex_comboBox">
         <item>
          <property name="text">
           <string>男</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>女</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="age_spinBox">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mail_lineedit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFontComboBox" name="fontComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>List:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="show_lineedit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pre_button">
       <property name="text">
        <string>Pre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_button">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
